test/e2e/resources/ca_hierarchy.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: acmpca.services.k8s.aws/v1alpha1
```

Comment:
Can we extend this to include a subordinate CA?

Reply:
The tests aren't using this file anymore, I'll delete it